client/components/popover/index.jsx

```diff
@@ -133,8 +133,11 @@ class Popover extends Component {
 			return null;
 		}
 
-		this.debug( 'Update position after inject DOM' );
-		this.setPosition();
+		this.debug( 'Update position after render completes' );
+
+		setTimeout( () => {
```

Comment:
This might be shortened to:
`setTimeout( () => this.setPosition(), 0 );`

Reply:
Thank you @tyxla for reviewing. Addressed. Let's 🚢